source/reference/verification/hash.txt

```diff
@@ -0,0 +1,98 @@
+.. _c2c-verify-hash-comp:
+
+===========================
+Verify with Hash Comparison
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can verify sync by comparing MD5 hashes of collections synced from the
+source cluster to the destination cluster. While hash comparison ensures
+that the destination cluster has received all changes from the source, the
+:dbcommand:`dbHash` command locks the cluster, preventing additional writes
+until it completes.
+
+.. note::
```

Comment:
Maybe take this blurb out of the admonition and put it in an About this Task section?

Reply:
Fixed